src/core_init_atmosphere/mpas_init_atm_thompson_aerosols.F

```diff
@@ -771,8 +771,8 @@ subroutine init_atm_thompson_aerosols_lbc(timestamp,timestart,block,mesh,diag,st
  call mpas_pool_get_dimension(mesh,'nVertLevels'  ,nVertLevels  )
  call mpas_pool_get_dimension(mesh,'nMonths'      ,nMonths      )
 
- call mpas_pool_get_dimension(state,'index_nifa',index_nifa)
```

Comment:
This is a change to NCAR's code. I emailed Laura Fowler about this change.

Reply:
Are you hoping to hear back before this is merged?

Reply:
This has now been merged into NCAR's 8.2.3:
https://github.com/MPAS-Dev/MPAS-Model/pull/1249